src/main/java/org/apache/sysds/runtime/instructions/Instruction.java

```diff
@@ -38,7 +41,21 @@
 		FEDERATED
 	}
 	
-	private static final Log LOG = LogFactory.getLog(Instruction.class.getName());
+	protected static final Log LOG = LogFactory.getLog(Instruction.class.getName());
```

Comment:
I made this private to ensure that the LOG is located in the same file it is called from.
If you want to make use of it somewhere else make a new Log object in that file.

Reply:
Private is also good. I do not need the "protected" to be there.